File: ex02/ex02_04/ex02_04.js
function main() {
  	let address = "1, Main Str., ";
  	const region = "West District, ";
  	const city = "Taichung";

  	address = address + region;
  	address = address + city;
  	console.log(address);
  	console.log("Length of address string: " + address.length);
  	return 0;
}

main();

File: ex02/ex02_04/ex02_04.py
def main():
    address = "1, Main Str., "
    region = "West District, "
    city = "Taichung"
    address = address + region
    address = address + city
    print(address)
    print("Length of address string: " + str(len(address)))
    return 0

if __name__ == "__main__":
    main()